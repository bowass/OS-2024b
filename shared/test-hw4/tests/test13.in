11079
7436
8045
33398
42960
19297
34555
47897
65409
30311
18315
756
29644
20009
54274
45205
40769
62896
47173
58010
24910
60201
34345
48122
24950
29734
13121
28023
10240
18652
2092
32568
38570
16142
10101
30174
26460
24549
5738
56536
56103
55717
61813
55809
34790
25120
38310
39615
24598
63250
986
34662
29022
20347
23847
6214
33244
17035
21994
37093
51538
51222
43473
61107
34699
58331
17050
40773
8743
26609
57288
65437
13364
29872
51710
32910
58850
41513
36362
1278
1975
8386
46634
62070
20706
7848
27958
60852
41941
6590
450
49788
27273
52008
25251
2782
34672
28295
13924
8740
4893
11630
57759
27194
53784
64518
6146
16409
50041
31010
38153
55853
7448
46077
507
14587
12964
29461
57145
24908
39070
37853
26300
51690
32820
38088
53531
64237
35555
17455
24475
22635
57907
25464
57326
64206
28783
24469
14515
36212
48909
35211
13425
36985
37576
5331
19283
40594
37191
43388
41365
6830
23748
59372
29471
63345
8399
65375
22524
1159
44607
5830
26693
29038
21092
1632
16592
25063
43720
2578
25339
39659
25296
9959
21389
38086
30507
3004
57330
49245
20706
17877
20691
27038
15421
62990
46259
58907
52562
63867
38671
33316
33644
12251
43588
26751
32771
56467
26955
63309
21006
30893
61128
21349
156
25319
52224
29462
33023
10312
15680
16344
13505
1005
60320
31838
61296
26908
39122
20912
48647
52746
48986
36083
1835
20090
1526
51315
618
62869
50350
47732
26571
58045
8358
25332
53515
16617
20045
64168
48078
60991
29924
43045
2163
30864
3717
40421
19014
37068
25096
28842
27790
63915
31716
59398
8958
44007
58294
20314
48526
42247
24475
40446
21104
25224
54231
45309
1723
14868
61886
26778
15493
45364
26059
62122
16267
20556
43603
26566
48658
64519
32903
48939
64385
20543
27260
59525
17514
45857
34794
52093
53056
2833
44699
7846
27215
30938
31576
28236
52902
58651
42595
57805
19671
24653
47662
8173
3314
23532
1100
62781
32794
23924
58763
17548
61333
44921
57453
6290
28557
55541
48969
26571
4826
37638
47182
51861
37049
61084
28489
11715
49225
65498
16839
17727
50430
9452
9753
27155
2923
49543
28157
48776
24462
38841
59826
36110
46946
10359
58532
50701
47851
43062
50103
13266
24215
52226
33688
6681
19584
23671
11488
42520
64822
56835
25129
44272
2398
12305
5701
38952
63668
53711
16383
61063
45214
9417
9627
61729
31332
63620
48294
63009
55391
13109
38201
4902
54409
5873
40336
4933
51430
52414
56303
19021
11007
39665
52210
30460
21106
50984
23586
40038
42118
64120
328
29701
25394
14879
21401
22486
16710
14944
30102
4852
29532
28661
43216
30382
7950
52147
2274
1719
10257
21808
39419
18666
4897
52959
33218
36534
17677
41288
19448
42314
53528
43360
63034
5493
15504
25127
7676
37102
10398
42100
22526
28695
29273
18543
44635
22053
6959
147
62315
30164
34964
32130
4622
43195
62626
42536
5592
55488
9525
38099
12028
22861
56904
26575
32193
44131
40912
16266
50866
53772
11995
56008
28604
16122
24382
26917
36622
54549
15939
18714
13859
22053
2396
58398
20532
15173
49285
602
5434
43328
23620
45900
63503
61010