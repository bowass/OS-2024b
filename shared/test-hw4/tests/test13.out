Virtual address: 11079 Physical address: 71 Value: -47
Virtual address: 7436 Physical address: 268 Value: 0
Virtual address: 8045 Physical address: 621 Value: 0
Virtual address: 33398 Physical address: 886 Value: 32
Virtual address: 42960 Physical address: 1232 Value: 0
Virtual address: 19297 Physical address: 1377 Value: 0
Virtual address: 34555 Physical address: 1787 Value: -66
Virtual address: 47897 Physical address: 1817 Value: 0
Virtual address: 65409 Physical address: 2177 Value: 0
Virtual address: 30311 Physical address: 2407 Value: -103
Virtual address: 18315 Physical address: 2699 Value: -30
Virtual address: 756 Physical address: 3060 Value: 0
Virtual address: 29644 Physical address: 3276 Value: 0
Virtual address: 20009 Physical address: 3369 Value: 0
Virtual address: 54274 Physical address: 3586 Value: 53
Virtual address: 45205 Physical address: 3989 Value: 0
Virtual address: 40769 Physical address: 4161 Value: 0
Virtual address: 62896 Physical address: 4528 Value: 0
Virtual address: 47173 Physical address: 4677 Value: 0
Virtual address: 58010 Physical address: 5018 Value: 56
Virtual address: 24910 Physical address: 5198 Value: 24
Virtual address: 60201 Physical address: 5417 Value: 0
Virtual address: 34345 Physical address: 1577 Value: 0
Virtual address: 48122 Physical address: 2042 Value: 46
Virtual address: 24950 Physical address: 5238 Value: 24
Virtual address: 29734 Physical address: 5670 Value: 29
Virtual address: 13121 Physical address: 5953 Value: 0
Virtual address: 28023 Physical address: 6263 Value: 93
Virtual address: 10240 Physical address: 6400 Value: 0
Virtual address: 18652 Physical address: 6876 Value: 0
Virtual address: 2092 Physical address: 6956 Value: 0
Virtual address: 32568 Physical address: 7224 Value: 0
Virtual address: 38570 Physical address: 7594 Value: 37
Virtual address: 16142 Physical address: 7694 Value: 15
Virtual address: 10101 Physical address: 8053 Value: 0
Virtual address: 30174 Physical address: 8414 Value: 29
Virtual address: 26460 Physical address: 8540 Value: 0
Virtual address: 24549 Physical address: 8933 Value: 0
Virtual address: 5738 Physical address: 9066 Value: 5
Virtual address: 56536 Physical address: 9432 Value: 0
Virtual address: 56103 Physical address: 9511 Value: -55
Virtual address: 55717 Physical address: 9893 Value: 0
Virtual address: 61813 Physical address: 10101 Value: 0
Virtual address: 55809 Physical address: 10241 Value: 0
Virtual address: 34790 Physical address: 10726 Value: 33
Virtual address: 25120 Physical address: 10784 Value: 0
Virtual address: 38310 Physical address: 11174 Value: 37
Virtual address: 39615 Physical address: 11455 Value: -81
Virtual address: 24598 Physical address: 11542 Value: 24
Virtual address: 63250 Physical address: 11794 Value: 61
Virtual address: 986 Physical address: 12250 Value: 0
Virtual address: 34662 Physical address: 10598 Value: 33
Virtual address: 29022 Physical address: 12382 Value: 28
Virtual address: 20347 Physical address: 12667 Value: -34
Virtual address: 23847 Physical address: 12839 Value: 73
Virtual address: 6214 Physical address: 13126 Value: 6
Virtual address: 33244 Physical address: 13532 Value: 0
Virtual address: 17035 Physical address: 13707 Value: -94
Virtual address: 21994 Physical address: 14058 Value: 21
Virtual address: 37093 Physical address: 14309 Value: 0
Virtual address: 51538 Physical address: 14418 Value: 50
Virtual address: 51222 Physical address: 14614 Value: 50
Virtual address: 43473 Physical address: 15057 Value: 0
Virtual address: 61107 Physical address: 15283 Value: -84
Virtual address: 34699 Physical address: 10635 Value: -30
Virtual address: 58331 Physical address: 15579 Value: -10
Virtual address: 17050 Physical address: 13722 Value: 16
Virtual address: 40773 Physical address: 4165 Value: 0
Virtual address: 8743 Physical address: 15655 Value: -119
Virtual address: 26609 Physical address: 8689 Value: 0
Virtual address: 57288 Physical address: 16072 Value: 0
Virtual address: 65437 Physical address: 2205 Value: 0
Virtual address: 13364 Physical address: 16180 Value: 0
Virtual address: 29872 Physical address: 5808 Value: 0
Virtual address: 51710 Physical address: 14590 Value: 50
Virtual address: 32910 Physical address: 16526 Value: 32
Virtual address: 58850 Physical address: 16866 Value: 57
Virtual address: 41513 Physical address: 16937 Value: 0
Virtual address: 36362 Physical address: 17162 Value: 35
Virtual address: 1278 Physical address: 17662 Value: 1
Virtual address: 1975 Physical address: 17847 Value: -19
Virtual address: 8386 Physical address: 18114 Value: 8
Virtual address: 46634 Physical address: 18218 Value: 45
Virtual address: 62070 Physical address: 18550 Value: 60
Virtual address: 20706 Physical address: 18914 Value: 20
Virtual address: 7848 Physical address: 19112 Value: 0
Virtual address: 27958 Physical address: 6198 Value: 27
Virtual address: 60852 Physical address: 19380 Value: 0
Virtual address: 41941 Physical address: 19669 Value: 0
Virtual address: 6590 Physical address: 19902 Value: 6
Virtual address: 450 Physical address: 20162 Value: 0
Virtual address: 49788 Physical address: 20348 Value: 0
Virtual address: 27273 Physical address: 20617 Value: 0
Virtual address: 52008 Physical address: 20776 Value: 0
Virtual address: 25251 Physical address: 10915 Value: -88
Virtual address: 2782 Physical address: 21214 Value: 2
Virtual address: 34672 Physical address: 10608 Value: 0
Virtual address: 28295 Physical address: 21383 Value: -95
Virtual address: 13924 Physical address: 21604 Value: 0
Virtual address: 8740 Physical address: 15652 Value: 0
Virtual address: 4893 Physical address: 21789 Value: 0
Virtual address: 11630 Physical address: 22126 Value: 11
Virtual address: 57759 Physical address: 22431 Value: 103
Virtual address: 27194 Physical address: 20538 Value: 26
Virtual address: 53784 Physical address: 22552 Value: 0
Virtual address: 64518 Physical address: 22790 Value: 63
Virtual address: 6146 Physical address: 13058 Value: 6
Virtual address: 16409 Physical address: 23065 Value: 0
Virtual address: 50041 Physical address: 23417 Value: 0
Virtual address: 31010 Physical address: 23586 Value: 30
Virtual address: 38153 Physical address: 11017 Value: 0
Virtual address: 55853 Physical address: 10285 Value: 0
Virtual address: 7448 Physical address: 280 Value: 0
Virtual address: 46077 Physical address: 24061 Value: 0
Virtual address: 507 Physical address: 20219 Value: 126
Virtual address: 14587 Physical address: 24315 Value: 62
Virtual address: 12964 Physical address: 24484 Value: 0
Virtual address: 29461 Physical address: 3093 Value: 0
Virtual address: 57145 Physical address: 15929 Value: 0
Virtual address: 24908 Physical address: 5196 Value: 0
Virtual address: 39070 Physical address: 24734 Value: 38
Virtual address: 37853 Physical address: 25053 Value: 0
Virtual address: 26300 Physical address: 25276 Value: 0
Virtual address: 51690 Physical address: 14570 Value: 50
Virtual address: 32820 Physical address: 16436 Value: 0
Virtual address: 38088 Physical address: 25544 Value: 0
Virtual address: 53531 Physical address: 25627 Value: 70
Virtual address: 64237 Physical address: 26093 Value: 0
Virtual address: 35555 Physical address: 26339 Value: -72
Virtual address: 17455 Physical address: 26415 Value: 11
Virtual address: 24475 Physical address: 8859 Value: -26
Virtual address: 22635 Physical address: 26731 Value: 26
Virtual address: 57907 Physical address: 4915 Value: -116
Virtual address: 25464 Physical address: 27000 Value: 0
Virtual address: 57326 Physical address: 16110 Value: 55
Virtual address: 64206 Physical address: 26062 Value: 62
Virtual address: 28783 Physical address: 27247 Value: 27
Virtual address: 24469 Physical address: 8853 Value: 0
Virtual address: 14515 Physical address: 24243 Value: 44
Virtual address: 36212 Physical address: 27508 Value: 0
Virtual address: 48909 Physical address: 27661 Value: 0
Virtual address: 35211 Physical address: 28043 Value: 98
Virtual address: 13425 Physical address: 16241 Value: 0
Virtual address: 36985 Physical address: 14201 Value: 0
Virtual address: 37576 Physical address: 28360 Value: 0
Virtual address: 5331 Physical address: 28627 Value: 52
Virtual address: 19283 Physical address: 1363 Value: -44
Virtual address: 40594 Physical address: 28818 Value: 39
Virtual address: 37191 Physical address: 28999 Value: 81
Virtual address: 43388 Physical address: 14972 Value: 0
Virtual address: 41365 Physical address: 29333 Value: 0
Virtual address: 6830 Physical address: 29614 Value: 6
Virtual address: 23748 Physical address: 29892 Value: 0
Virtual address: 59372 Physical address: 30188 Value: 0
Virtual address: 29471 Physical address: 3103 Value: -57
Virtual address: 63345 Physical address: 11889 Value: 0
Virtual address: 8399 Physical address: 18127 Value: 51
Virtual address: 65375 Physical address: 2143 Value: -41
Virtual address: 22524 Physical address: 30460 Value: 0
Virtual address: 1159 Physical address: 17543 Value: 33
Virtual address: 44607 Physical address: 30527 Value: -113
Virtual address: 5830 Physical address: 9158 Value: 5
Virtual address: 26693 Physical address: 30789 Value: 0
Virtual address: 29038 Physical address: 12398 Value: 28
Virtual address: 21092 Physical address: 31076 Value: 0
Virtual address: 1632 Physical address: 31328 Value: 0
Virtual address: 16592 Physical address: 23248 Value: 0
Virtual address: 25063 Physical address: 5351 Value: 121
Virtual address: 43720 Physical address: 31688 Value: 0
Virtual address: 2578 Physical address: 21010 Value: 2
Virtual address: 25339 Physical address: 11003 Value: -66
Virtual address: 39659 Physical address: 11499 Value: -70
Virtual address: 25296 Physical address: 10960 Value: 0
Virtual address: 9959 Physical address: 31975 Value: -71
Virtual address: 21389 Physical address: 32141 Value: 0
Virtual address: 38086 Physical address: 25542 Value: 37
Virtual address: 30507 Physical address: 32299 Value: -54
Virtual address: 3004 Physical address: 32700 Value: 0
Virtual address: 57330 Physical address: 16114 Value: 55
Virtual address: 49245 Physical address: 32861 Value: 0
Virtual address: 20706 Physical address: 18914 Value: 20
Virtual address: 17877 Physical address: 33237 Value: 0
Virtual address: 20691 Physical address: 18899 Value: 52
Virtual address: 27038 Physical address: 33438 Value: 26
Virtual address: 15421 Physical address: 33597 Value: 0
Virtual address: 62990 Physical address: 33806 Value: 61
Virtual address: 46259 Physical address: 34227 Value: 44
Virtual address: 58907 Physical address: 34331 Value: -122
Virtual address: 52562 Physical address: 34642 Value: 51
Virtual address: 63867 Physical address: 34939 Value: 94
Virtual address: 38671 Physical address: 35087 Value: -61
Virtual address: 33316 Physical address: 804 Value: 0
Virtual address: 33644 Physical address: 35436 Value: 0
Virtual address: 12251 Physical address: 35803 Value: -10
Virtual address: 43588 Physical address: 31556 Value: 0
Virtual address: 26751 Physical address: 30847 Value: 31
Virtual address: 32771 Physical address: 16387 Value: 0
Virtual address: 56467 Physical address: 9363 Value: 36
Virtual address: 26955 Physical address: 33355 Value: 82
Virtual address: 63309 Physical address: 11853 Value: 0
Virtual address: 21006 Physical address: 30990 Value: 20
Virtual address: 30893 Physical address: 36013 Value: 0
Virtual address: 61128 Physical address: 15304 Value: 0
Virtual address: 21349 Physical address: 32101 Value: 0
Virtual address: 156 Physical address: 36252 Value: 0
Virtual address: 25319 Physical address: 10983 Value: -71
Virtual address: 52224 Physical address: 36352 Value: 0
Virtual address: 29462 Physical address: 3094 Value: 28
Virtual address: 33023 Physical address: 16639 Value: 63
Virtual address: 10312 Physical address: 6472 Value: 0
Virtual address: 15680 Physical address: 36672 Value: 0
Virtual address: 16344 Physical address: 7896 Value: 0
Virtual address: 13505 Physical address: 16321 Value: 0
Virtual address: 1005 Physical address: 12269 Value: 0
Virtual address: 60320 Physical address: 5536 Value: 0
Virtual address: 31838 Physical address: 36958 Value: 31
Virtual address: 61296 Physical address: 37232 Value: 0
Virtual address: 26908 Physical address: 33308 Value: 0
Virtual address: 39122 Physical address: 24786 Value: 38
Virtual address: 20912 Physical address: 37552 Value: 0
Virtual address: 48647 Physical address: 37639 Value: -127
Virtual address: 52746 Physical address: 37898 Value: 51
Virtual address: 48986 Physical address: 27738 Value: 47
Virtual address: 36083 Physical address: 38387 Value: 60
Virtual address: 1835 Physical address: 17707 Value: -54
Virtual address: 20090 Physical address: 3450 Value: 19
Virtual address: 1526 Physical address: 38646 Value: 1
Virtual address: 51315 Physical address: 14707 Value: 28
Virtual address: 618 Physical address: 2922 Value: 0
Virtual address: 62869 Physical address: 4501 Value: 0
Virtual address: 50350 Physical address: 38830 Value: 49
Virtual address: 47732 Physical address: 39028 Value: 0
Virtual address: 26571 Physical address: 8651 Value: -14
Virtual address: 58045 Physical address: 5053 Value: 0
Virtual address: 8358 Physical address: 18086 Value: 8
Virtual address: 25332 Physical address: 10996 Value: 0
Virtual address: 53515 Physical address: 25611 Value: 66
Virtual address: 16617 Physical address: 23273 Value: 0
Virtual address: 20045 Physical address: 3405 Value: 0
Virtual address: 64168 Physical address: 26024 Value: 0
Virtual address: 48078 Physical address: 1998 Value: 46
Virtual address: 60991 Physical address: 15167 Value: -113
Virtual address: 29924 Physical address: 5860 Value: 0
Virtual address: 43045 Physical address: 39205 Value: 0
Virtual address: 2163 Physical address: 7027 Value: 28
Virtual address: 30864 Physical address: 35984 Value: 0
Virtual address: 3717 Physical address: 39557 Value: 0
Virtual address: 40421 Physical address: 39909 Value: 0
Virtual address: 19014 Physical address: 40006 Value: 18
Virtual address: 37068 Physical address: 14284 Value: 0
Virtual address: 25096 Physical address: 10760 Value: 0
Virtual address: 28842 Physical address: 27306 Value: 28
Virtual address: 27790 Physical address: 40334 Value: 27
Virtual address: 63915 Physical address: 34987 Value: 106
Virtual address: 31716 Physical address: 40676 Value: 0
Virtual address: 59398 Physical address: 40710 Value: 58
Virtual address: 8958 Physical address: 15870 Value: 8
Virtual address: 44007 Physical address: 41191 Value: -7
Virtual address: 58294 Physical address: 15542 Value: 56
Virtual address: 20314 Physical address: 12634 Value: 19
Virtual address: 48526 Physical address: 41358 Value: 47
Virtual address: 42247 Physical address: 41479 Value: 65
Virtual address: 24475 Physical address: 8859 Value: -26
Virtual address: 40446 Physical address: 39934 Value: 39
Virtual address: 21104 Physical address: 31088 Value: 0
Virtual address: 25224 Physical address: 10888 Value: 0
Virtual address: 54231 Physical address: 41943 Value: -11
Virtual address: 45309 Physical address: 4093 Value: 0
Virtual address: 1723 Physical address: 31419 Value: -82
Virtual address: 14868 Physical address: 42004 Value: 0
Virtual address: 61886 Physical address: 10174 Value: 60
Virtual address: 26778 Physical address: 30874 Value: 26
Virtual address: 15493 Physical address: 33669 Value: 0
Virtual address: 45364 Physical address: 42292 Value: 0
Virtual address: 26059 Physical address: 42699 Value: 114
Virtual address: 62122 Physical address: 18602 Value: 60
Virtual address: 16267 Physical address: 7819 Value: -30
Virtual address: 20556 Physical address: 18764 Value: 0
Virtual address: 43603 Physical address: 31571 Value: -108
Virtual address: 26566 Physical address: 8646 Value: 25
Virtual address: 48658 Physical address: 37650 Value: 47
Virtual address: 64519 Physical address: 22791 Value: 1
Virtual address: 32903 Physical address: 16519 Value: 33
Virtual address: 48939 Physical address: 27691 Value: -54
Virtual address: 64385 Physical address: 42881 Value: 0
Virtual address: 20543 Physical address: 18751 Value: 15
Virtual address: 27260 Physical address: 20604 Value: 0
Virtual address: 59525 Physical address: 40837 Value: 0
Virtual address: 17514 Physical address: 26474 Value: 17
Virtual address: 45857 Physical address: 23841 Value: 0
Virtual address: 34794 Physical address: 10730 Value: 33
Virtual address: 52093 Physical address: 20861 Value: 0
Virtual address: 53056 Physical address: 43072 Value: 0
Virtual address: 2833 Physical address: 32529 Value: 0
Virtual address: 44699 Physical address: 30619 Value: -90
Virtual address: 7846 Physical address: 19110 Value: 7
Virtual address: 27215 Physical address: 20559 Value: -109
Virtual address: 30938 Physical address: 36058 Value: 30
Virtual address: 31576 Physical address: 40536 Value: 0
Virtual address: 28236 Physical address: 21324 Value: 0
Virtual address: 52902 Physical address: 38054 Value: 51
Virtual address: 58651 Physical address: 16667 Value: 70
Virtual address: 42595 Physical address: 43363 Value: -104
Virtual address: 57805 Physical address: 22477 Value: 0
Virtual address: 19671 Physical address: 43735 Value: 53
Virtual address: 24653 Physical address: 11597 Value: 0
Virtual address: 47662 Physical address: 38958 Value: 46
Virtual address: 8173 Physical address: 749 Value: 0
Virtual address: 3314 Physical address: 44018 Value: 3
Virtual address: 23532 Physical address: 44268 Value: 0
Virtual address: 1100 Physical address: 17484 Value: 0
Virtual address: 62781 Physical address: 4413 Value: 0
Virtual address: 32794 Physical address: 16410 Value: 32
Virtual address: 23924 Physical address: 12916 Value: 0
Virtual address: 58763 Physical address: 16779 Value: 98
Virtual address: 17548 Physical address: 26508 Value: 0
Virtual address: 61333 Physical address: 37269 Value: 0
Virtual address: 44921 Physical address: 44409 Value: 0
Virtual address: 57453 Physical address: 44653 Value: 0
Virtual address: 6290 Physical address: 13202 Value: 6
Virtual address: 28557 Physical address: 44941 Value: 0
Virtual address: 55541 Physical address: 45301 Value: 0
Virtual address: 48969 Physical address: 27721 Value: 0
Virtual address: 26571 Physical address: 8651 Value: -14
Virtual address: 4826 Physical address: 45530 Value: 4
Virtual address: 37638 Physical address: 24838 Value: 36
Virtual address: 47182 Physical address: 4686 Value: 46
Virtual address: 51861 Physical address: 45717 Value: 0
Virtual address: 37049 Physical address: 14265 Value: 0
Virtual address: 61084 Physical address: 15260 Value: 0
Virtual address: 28489 Physical address: 44873 Value: 0
Virtual address: 11715 Physical address: 22211 Value: 112
Virtual address: 49225 Physical address: 32841 Value: 0
Virtual address: 65498 Physical address: 2266 Value: 63
Virtual address: 16839 Physical address: 46023 Value: 113
Virtual address: 17727 Physical address: 33087 Value: 79
Virtual address: 50430 Physical address: 38910 Value: 49
Virtual address: 9452 Physical address: 46316 Value: 0
Virtual address: 9753 Physical address: 31769 Value: 0
Virtual address: 27155 Physical address: 20499 Value: -124
Virtual address: 2923 Physical address: 32619 Value: -38
Virtual address: 49543 Physical address: 46471 Value: 97
Virtual address: 28157 Physical address: 6397 Value: 0
Virtual address: 48776 Physical address: 37768 Value: 0
Virtual address: 24462 Physical address: 8846 Value: 23
Virtual address: 38841 Physical address: 35257 Value: 0
Virtual address: 59826 Physical address: 46770 Value: 58
Virtual address: 36110 Physical address: 27406 Value: 35
Virtual address: 46946 Physical address: 46946 Value: 45
Virtual address: 10359 Physical address: 6519 Value: 29
Virtual address: 58532 Physical address: 47268 Value: 0
Virtual address: 50701 Physical address: 47373 Value: 0
Virtual address: 47851 Physical address: 39147 Value: -70
Virtual address: 43062 Physical address: 39222 Value: 42
Virtual address: 50103 Physical address: 23479 Value: -19
Virtual address: 13266 Physical address: 6098 Value: 12
Virtual address: 24215 Physical address: 47767 Value: -91
Virtual address: 52226 Physical address: 36354 Value: 51
Virtual address: 33688 Physical address: 35480 Value: 0
Virtual address: 6681 Physical address: 29465 Value: 0
Virtual address: 19584 Physical address: 43648 Value: 0
Virtual address: 23671 Physical address: 29815 Value: 29
Virtual address: 11488 Physical address: 48096 Value: 0
Virtual address: 42520 Physical address: 43288 Value: 0
Virtual address: 64822 Physical address: 48182 Value: 63
Virtual address: 56835 Physical address: 48387 Value: -128
Virtual address: 25129 Physical address: 10793 Value: 0
Virtual address: 44272 Physical address: 48880 Value: 0
Virtual address: 2398 Physical address: 48990 Value: 2
Virtual address: 12305 Physical address: 49169 Value: 0
Virtual address: 5701 Physical address: 9029 Value: 0
Virtual address: 38952 Physical address: 24616 Value: 0
Virtual address: 63668 Physical address: 49588 Value: 0
Virtual address: 53711 Physical address: 25807 Value: 115
Virtual address: 16383 Physical address: 7935 Value: -1
Virtual address: 61063 Physical address: 15239 Value: -95
Virtual address: 45214 Physical address: 3998 Value: 44
Virtual address: 9417 Physical address: 46281 Value: 0
Virtual address: 9627 Physical address: 49819 Value: 102
Virtual address: 61729 Physical address: 10017 Value: 0
Virtual address: 31332 Physical address: 50020 Value: 0
Virtual address: 63620 Physical address: 49540 Value: 0
Virtual address: 48294 Physical address: 50342 Value: 47
Virtual address: 63009 Physical address: 33825 Value: 0
Virtual address: 55391 Physical address: 45151 Value: 23
Virtual address: 13109 Physical address: 5941 Value: 0
Virtual address: 38201 Physical address: 11065 Value: 0
Virtual address: 4902 Physical address: 21798 Value: 4
Virtual address: 54409 Physical address: 3721 Value: 0
Virtual address: 5873 Physical address: 9201 Value: 0
Virtual address: 40336 Physical address: 39824 Value: 0
Virtual address: 4933 Physical address: 21829 Value: 0
Virtual address: 51430 Physical address: 14822 Value: 50
Virtual address: 52414 Physical address: 36542 Value: 51
Virtual address: 56303 Physical address: 9711 Value: -5
Virtual address: 19021 Physical address: 40013 Value: 0
Virtual address: 11007 Physical address: 50687 Value: -65
Virtual address: 39665 Physical address: 11505 Value: 0
Virtual address: 52210 Physical address: 20978 Value: 50
Virtual address: 30460 Physical address: 2556 Value: 0
Virtual address: 21106 Physical address: 31090 Value: 20
Virtual address: 50984 Physical address: 50728 Value: 0
Virtual address: 23586 Physical address: 29730 Value: 23
Virtual address: 40038 Physical address: 51046 Value: 39
Virtual address: 42118 Physical address: 51334 Value: 41
Virtual address: 64120 Physical address: 25976 Value: 0
Virtual address: 328 Physical address: 20040 Value: 0
Virtual address: 29701 Physical address: 5637 Value: 0
Virtual address: 25394 Physical address: 26930 Value: 24
Virtual address: 14879 Physical address: 42015 Value: -121
Virtual address: 21401 Physical address: 32153 Value: 0
Virtual address: 22486 Physical address: 30422 Value: 21
Virtual address: 16710 Physical address: 45894 Value: 16
Virtual address: 14944 Physical address: 42080 Value: 0
Virtual address: 30102 Physical address: 8342 Value: 29
Virtual address: 4852 Physical address: 45556 Value: 0
Virtual address: 29532 Physical address: 3164 Value: 0
Virtual address: 28661 Physical address: 45045 Value: 0
Virtual address: 43216 Physical address: 39376 Value: 0
Virtual address: 30382 Physical address: 2478 Value: 29
Virtual address: 7950 Physical address: 526 Value: 7
Virtual address: 52147 Physical address: 20915 Value: -20
Virtual address: 2274 Physical address: 7138 Value: 2
Virtual address: 1719 Physical address: 31415 Value: -83
Virtual address: 10257 Physical address: 6417 Value: 0
Virtual address: 21808 Physical address: 13872 Value: 0
Virtual address: 39419 Physical address: 51707 Value: 126
Virtual address: 18666 Physical address: 6890 Value: 18
Virtual address: 4897 Physical address: 21793 Value: 0
Virtual address: 52959 Physical address: 38111 Value: -73
Virtual address: 33218 Physical address: 13506 Value: 32
Virtual address: 36534 Physical address: 17334 Value: 35
Virtual address: 17677 Physical address: 33037 Value: 0
Virtual address: 41288 Physical address: 29256 Value: 0
Virtual address: 19448 Physical address: 1528 Value: 0
Virtual address: 42314 Physical address: 41546 Value: 41
Virtual address: 53528 Physical address: 25624 Value: 0
Virtual address: 43360 Physical address: 14944 Value: 0
Virtual address: 63034 Physical address: 33850 Value: 61
Virtual address: 5493 Physical address: 51829 Value: 0
Virtual address: 15504 Physical address: 33680 Value: 0
Virtual address: 25127 Physical address: 10791 Value: -119
Virtual address: 7676 Physical address: 508 Value: 0
Virtual address: 37102 Physical address: 14318 Value: 36
Virtual address: 10398 Physical address: 6558 Value: 10
Virtual address: 42100 Physical address: 51316 Value: 0
Virtual address: 22526 Physical address: 30462 Value: 21
Virtual address: 28695 Physical address: 27159 Value: 5
Virtual address: 29273 Physical address: 52057 Value: 0
Virtual address: 18543 Physical address: 6767 Value: 27
Virtual address: 44635 Physical address: 30555 Value: -106
Virtual address: 22053 Physical address: 52261 Value: 0
Virtual address: 6959 Physical address: 52527 Value: -53
Virtual address: 147 Physical address: 36243 Value: 36
Virtual address: 62315 Physical address: 52843 Value: -38
Virtual address: 30164 Physical address: 8404 Value: 0
Virtual address: 34964 Physical address: 53140 Value: 0
Virtual address: 32130 Physical address: 53378 Value: 31
Virtual address: 4622 Physical address: 45326 Value: 4
Virtual address: 43195 Physical address: 39355 Value: 46
Virtual address: 62626 Physical address: 53666 Value: 61
Virtual address: 42536 Physical address: 43304 Value: 0
Virtual address: 5592 Physical address: 51928 Value: 0
Virtual address: 55488 Physical address: 45248 Value: 0
Virtual address: 9525 Physical address: 49717 Value: 0
Virtual address: 38099 Physical address: 25555 Value: 52
Virtual address: 12028 Physical address: 54012 Value: 0
Virtual address: 22861 Physical address: 54093 Value: 0
Virtual address: 56904 Physical address: 48456 Value: 0
Virtual address: 26575 Physical address: 8655 Value: -13
Virtual address: 32193 Physical address: 53441 Value: 0
Virtual address: 44131 Physical address: 48739 Value: 24
Virtual address: 40912 Physical address: 4304 Value: 0
Virtual address: 16266 Physical address: 7818 Value: 15
Virtual address: 50866 Physical address: 47538 Value: 49
Virtual address: 53772 Physical address: 22540 Value: 0
Virtual address: 11995 Physical address: 53979 Value: -74
Virtual address: 56008 Physical address: 10440 Value: 0
Virtual address: 28604 Physical address: 44988 Value: 0
Virtual address: 16122 Physical address: 54522 Value: 15
Virtual address: 24382 Physical address: 8766 Value: 23
Virtual address: 26917 Physical address: 33317 Value: 0
Virtual address: 36622 Physical address: 54542 Value: 35
Virtual address: 54549 Physical address: 54805 Value: 0
Virtual address: 15939 Physical address: 54339 Value: -112
Virtual address: 18714 Physical address: 55066 Value: 18
Virtual address: 13859 Physical address: 21539 Value: -120
Virtual address: 22053 Physical address: 52261 Value: 0
Virtual address: 2396 Physical address: 48988 Value: 0
Virtual address: 58398 Physical address: 47134 Value: 57
Virtual address: 20532 Physical address: 18740 Value: 0
Virtual address: 15173 Physical address: 55365 Value: 0
Virtual address: 49285 Physical address: 32901 Value: 0
Virtual address: 602 Physical address: 2906 Value: 0
Virtual address: 5434 Physical address: 51770 Value: 5
Virtual address: 43328 Physical address: 14912 Value: 0
Virtual address: 23620 Physical address: 29764 Value: 0
Virtual address: 45900 Physical address: 23884 Value: 0
Virtual address: 63503 Physical address: 49423 Value: 3
Virtual address: 61010 Physical address: 15186 Value: 59
Number of Translated Addresses = 500
Page Faults = 217
Page Fault Rate = 0.434
TLB Hits = 27
TLB Hit Rate = 0.054
